Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -913,12 +880,16 @@ extension FirefoxHomeViewController {
         return cell
     }
 
-    func configurePocketItemCell(_ cell: UICollectionViewCell, forIndexPath indexPath: IndexPath) -> UICollectionViewCell {
-        let pocketStory = pocketStories[indexPath.row]
-        let pocketItemCell = cell as! FirefoxHomeHighlightCell
-        pocketItemCell.configureWithPocketStory(pocketStory)
+    private func configurePocketItemCell(_ cell: UICollectionViewCell, forIndexPath indexPath: IndexPath) -> UICollectionViewCell {
+        pocketViewModel.recordSectionHasShown()
 
-        return pocketItemCell
+        let pocketCell = cell as! FxHomePocketCollectionCell
```

Comment:
hmm force unwrap 🤔

Reply:
We do that in each of the section. We could put a guard instead, but I would do it for all the sections